packages/components/src/SegmentedControl/SegmentedControl.tsx

```diff
@@ -18,21 +18,32 @@ interface SegmentedControlProps<T> extends PropsWithChildren {
    * The default option to be selected initially
    */
   readonly defaultValue?: T;
+
+  /**
+   * A unique name for the SegmentedControl, that links the group of
+   * options together.
+   *
+   * @default useId()
+   */
+  readonly name?: string;
 }
 
 export function SegmentedControl<T>({
   selectedValue,
   onSelectValue,
   defaultValue,
   children,
+  name = useId(),
 }: SegmentedControlProps<T>) {
   const container = useRef<HTMLDivElement>(null);
+  console.log("name", name);
```

Comment:
There are a couple of `console.log` calls to remove 😄

Reply:
classic! removed them here 022a54257396fbcbe86a1e4793737bab2085f7cb